redis/src/main/java/org/hibernate/ogm/datastore/redis/impl/json/JsonSerializationStrategy.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.redis.impl.json;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.hibernate.ogm.datastore.redis.impl.SerializationStrategy;
+import org.hibernate.ogm.type.spi.GridType;
+import org.hibernate.type.Type;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Serialize payload as JSON using the Jackson Mapper.
+ *
+ * @author Mark Paluch
+ */
+public class JsonSerializationStrategy implements SerializationStrategy {
```

Comment:
What other strategy impls are there besides this one? I'd only establish an interface once there are actual several impls.